api/searchCases/index.ts

```diff
@@ -1,137 +1,171 @@
-import { caseMetaDataFiledsMapping } from '../configuration/mappings'
-import { fieldNameMapper } from '../lib/util'
+import {UserInfo} from '../auth/interfaces/UserInfo'
```

Comment:
Going forward, can we make sure imports conform to the standardised pattern of having spaces inside the curly brackets, please? 😉